ratatui-core/Cargo.toml

```diff
@@ -77,6 +76,7 @@ unicode-truncate = { workspace = true, default-features = false }
 unicode-width.workspace = true
 
 [dev-dependencies]
+indoc.workspace = true
```

Comment:
Please provide another PR for this. I think we can merge this right away instead of making it sit here